index.html

```diff
@@ -186,73 +186,72 @@ <h3 class="mt-post">Multi-Post-Stories</h3>
-          </div>
-        </div>
-        <section class="second-section bg-white">
-          <div class="lang-dropdown">
-            <h3>Languages</h3>
-            <span class="dropdown"></span>
-          </div>
-          <div class="technologies">
-            <div class="tech">
-              <img src="images/icjscript.png" alt="dropdown" />
-              <p>Javascript</p>
+        <div id="section-5" class="contain5">
+          <div class="s5">
+            <div class="about-text">
+              <div class="head-text">
+                <h1>About Myself</h1>
+              </div>
+              <p>
+                Hello I’m a software developer! I can help you build a product ,
+                feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.
+              </p>
+            </div>
+            <div class="lc">LET'S CONNECT</div>
+
+            <div class="base-icon-5">
+              <ul>
+                <li>
+                  <a href=""><img src="images/Vector-1.svg" alt="image" /></a>
+                </li>
+                <li>
+                  <a href=""
+                    ><img src="images/Linkedinicon.svg" alt="image"
+                  /></a>
+                </li>
+                <li>
+                  <a href=""><img src="images/Vector-2.svg" alt="image" /></a>
+                </li>
+                <li>
+                  <a href=""><img src="images/Vector-3.svg" alt="image" /></a>
+                </li>
+                <li>
+                  <a href=""><img src="images/Vector.svg" alt="image" /></a>
+                </li>
+              </ul>
             </div>
-            <div class="tech">
-              <img src="images/ichtml.png" alt="dropdown" />
-              <p>Html</p>
+            <div class="resume">
+              <a href="">Get my resume</a>
             </div>
-            <div class="tech">
-              <img src="images/iccss.png" alt="dropdown" />
-              <p>CSS</p>
+            <div class="langv">
+              <h3>Languages</h3>
+              <img src="images/icdropdown.png" alt="icon" />
+            </div>
+            <div class="tech5">
+              <ul>
+                <li class="item1">
+                  <img src="images/icjscript.png" alt="image" />
+                </li>
+                <li class="item2">
+                  <img src="images/ichtml.png" alt="image" />
+                </li>
+                <li class="item3">
+                  <img src="images/iccss.png" alt="image" />
+                </li>
+              </ul>
+            </div>
+            <div class="frame">
+              <h3>Frameworks</h3>
+              <img src="images/Union.png" alt="icon" />
+            </div>
+            <div class="skill">
+              <h3>Skills</h3>
+              <img src="images/Union.png" alt="icon" />
             </div>
```

Comment:
- [ ] Good following what the previous reviewer mentioned, all that's left is to make all the categories as an unordered list `ul` and each category (Languages, Frameworks and Skills) should be inside an `li` tag, please do that. 😁